packages/overlay/src/App.tsx

```diff
@@ -217,10 +218,10 @@ export default function App({
   useEffect(() => {
     if (!isOpen) {
       spotlightEventTarget.dispatchEvent(new CustomEvent('closed'));
-      document.body.style.overflow = 'auto';
```

Comment:
We just need to remove this, right? 

But I think it's good to have another stylesheet layer added with classname

Reply:
Removing wouldn't fix as we need a way of undoing our "hidden" override when a user toggles Spotlight closed